src/test/scala/chiseltest/simulator/VerilatorCirct.scala

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiseltest.simulator
+
+import chiseltest.simulator.jna.JNAUtils
+import chiseltest.utils.CaptureStdout
+import org.scalatest.Tag
+import chiseltest.utils.FlatSpecWithTargetDir
+
+/** To disable tests that require the commercial VCS simulator use the following: `sbt testOnly -- -l RequiresVerilator` */
+object RequiresVerilatorCirct extends Tag("RequiresVerilatorCirct")
```

Comment:
Instead of a `RequiresVerilatorCirct` tag, I would prefer if you could make a `RequiresFirTool` tag and then tag all tests as `RequiresVerilator` and `RequiresFirTool` since we might later start using the `firtool` for other Verilog simulators as well.

Reply:
@ekiwi I'm not sure which is the right way to do this. Would you add an extra argument to the whole `(BasicCompliance, StepCompliance, ...., StopAssertAssumeCompliance` family, in order to pass in multiple tabs?

Reply:
we can probably overload the constructor to accept a Seq of Tags

Reply:
@ekiwi I've looked at this a little bit and it seems do-able (i think) but difficult enough that I'd prefer to do this work under a different PR, the CIRCT stuff is undergoing a lot of work and this PR will enable me to use chiseltest+circt in more situations and discover